explainshell/web/static/js/es.js

```diff
@@ -1081,7 +1081,10 @@ function draw() {
 
 // Theme-related stuff
 $(document).ready(function() {
-    var selectedTheme = localStorage.getItem('theme') || 'default';
+    // use theme from local storage or auto-detect otherwise
+    var selectedTheme = localStorage.getItem('theme')
+        || (window.matchMedia("(prefers-color-scheme: dark)").matches ? 'dark' : 'default')
```

Comment:
First time I'm seeing [matchMedia](https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia#Browser_compatibility), but looks like it's everywhere.